ipn/ipnlocal/c2n.go

```diff
@@ -300,6 +300,13 @@ func handleC2NUpdatePost(b *LocalBackend, w http.ResponseWriter, r *http.Request
 		return
 	}
 
+	// Do not update if we have active inbound SSH connections. Control can set
+	// force=true query parameter to override this.
+	if r.FormValue("force") != "true" && b.sshServer.NumActiveConns() > 0 {
```

Comment:
b.sshServer  is documented as:

```go
	sshServer      SSHServer          // or nil, initialized lazily.
```

Need to handle `nil` here (and note it's a nil interface, so a method permitted a nil receiver won't work)

Reply:
good call, added a nil check